### DISCUSSION TITLE
Hire a developer

### DISCUSSION THREAD

**Discussion by obaviet** [saleor/saleor#8025] [Deployment]
Hello
I must admit that I am new to saleor and I know little about php. I watched the saleor demo and really liked it because of its amazing loading speed. However there are 2 problems that I need to solve to be able to use it as a product:
- I need to integrate a local payment gateway : https://developers.momo.vn/#/docs/en/
- I need to redesign Storefront beautifully
Is there a developer here that can collaborate with me?

**Comment by karolkielecki** [comment]
@obaviet Hi! Saleor is powered by Python, not PHP. You can find more info on payment gateways here: https://docs.saleor.io/docs/3.0/developer/extending/payment-gateways. 

Regarding Storefront--check the following thread: https://spectrum.chat/saleor/saleor-storefront/modifying-the-storefront~c1955dbf-a421-4fb6-b99e-937dd2642b23

**Reply by obaviet** [reply]
I have been trying to find developers to help me with this but it seems very difficult. There are too few developers

**Reply by Juleszz** [reply]
Maybe I can help you with this, just let me know how I can contact you best for further communication

**Reply by obaviet** [reply]
my fb : https://www.facebook.com/thiduyen.vang.16/

**Reply by Diego17ccs** [reply]
hello, did you solved ??

**Reply by CopperMS** [reply]
> Maybe I can help you with this, just let me know how I can contact you best for further communication

Hi Juleszz, I too am looking for a developer (on an existing Saleor installation). Please reply here or DM me (if possible), and I'll provide more details.
